code/game/machinery/buttons.dm

```diff
@@ -1,3 +1,111 @@
+#define DOOR_FLAG_OPEN_ONLY (1 << 0)
```

Comment:
Could you move this to the machine defines?

Reply:
It's being undefined at the end of the file. This flag won't likely survive it if we add assemblies and electronics as intended.
This is a very watered-down version of /tg/ buttons because a ton of stuff has been striped away.